xcore/Makefile.am

```diff
@@ -51,6 +51,7 @@ xcam_sources = \
     interface/blender.cpp               \
     interface/geo_mapper.cpp            \
     interface/stitcher.cpp              \
+    once_map_video_buffer_priv.cpp      \
```

Comment:
move once_map_video_buffer_priv.cpp together with other xcore cp files and keep interface/*.cpp separate.